npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -323,6 +323,37 @@ def self.run_npm_command(command, fingerprint: command)
         end
       end
 
+      sig { returns(T.nilable(String)) }
+      def self.node_version
```

Comment:
It looks like this function only gets called from a single location.  Will it be used more broadly in the future?  If not, is there another reason to add this as a helper and not define it next to where it's used in the package manager, or defined directly on the node language class?

Reply:
@jonabc  
While it’s currently used in one place, having it in a helper makes future updates easier if we need to retrieve the Node.js version differently (e.g., from the environment). It also centralizes logic that might be reused, especially for commands dependent on the Node.js version. Generally, we use helpers to manage command execution logic that may change across package managers.